packages/ag.rb

```diff
@@ -12,11 +12,12 @@ class Ag < Package
   binary_sha256 ({
   })
 
-  depends_on "autoconf"
-  depends_on "automake"
-  depends_on "pkgconfig"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkgconfig" => :build
   depends_on "pcre"
   depends_on "xzutils"
+  depends_on "zlibpkg"
 
   def self.build
     system "autoreconf", "-fiv"
```

Comment:
Do you know why this is necessary? Usually packagers should not run autoreconf.

Reply:
I'm guessing because of the autoreconf line.  @vince, can you please provide more details?

Reply:
I was asking specifically about the autoreconf line. I may have marked the wrong line.

Reply:
I largely took this recipe from the homebrew package. Could be it's not needed but I stopped fiddling when the package worked.